### PR TITLE
Moretests

### PR DESCRIPTION
**Pull request by chupman**
I added logic to handle multiple -s in a local files reference. Ran against community and didn't see any issues. I'll do some runs tomorrow in travis to get an average runtime.

### CODE REVIEW

**Comment by chupman** [comment]
I was just looking through the output logs and realized I forgot to replace the -'s with wildcards for the grep search. I'll try to fix it tomorrow.
../community/contributors/design-proposals/apparmor.md: Can't find anchor1 '#alpha-beta-and-stable-versions'
../community/contributors/design-proposals/apparmor.md: Can't find anchor1 '#api-changes'

**Comment by chupman** [comment]
I'll try to do some more testing tomorrow and take a look to see if I can improve the flow, but I think this should be generally considered functional.

**Comment by chupman** [comment]
@duglin I cleaned up the logic and it should be much easier to review now. 
Here's it run in travis against community https://travis-ci.org/chupman/community/builds/219912654

I think there are some false positives on the can't load warnings due to a lack of outbound connectivity in the container, since I'm not seeing those locally.

File: bin/verify-links.sh [hunk 1]
```diff
@@ -134,24 +134,46 @@ for file in ${mdFiles}; do
...
     fi
 
-    # Local file href - skip for now.
-    # TODO add support for checking these
-    if [ "${ref:0:1}" == "#" ]; then
-      ref=${ref:1}
-      if ! grep "^$ref$" ${tmp}anchors1 > /dev/null 2>&1 ; then
-        echo $file: Can\'t find anchor \'\#${ref}\' | tee -a ${tmp}3
+    # Local repo markdown link (i.e. ref contains a #) 
+    if [[ "${ref/\#}" != "${ref}" ]]; then
```
Comment (duglin) [inline bin/verify-links.sh:138]: One cool thing about this work is that I'm learning (or remembering) way more about sh/bash stuff than I would normally - e.g. all of this variable expansion/substitution stuff.

File: bin/verify-links.sh [hunk 2]
```diff
@@ -134,24 +134,46 @@ for file in ${mdFiles}; do
...
+      # Check to see if the link starts with a hash, if not we need to update the filepath
```
Comment (duglin) [inline bin/verify-links.sh:139]: Minor thing, but the indent of the comment above is off

Comment (duglin) [reply bin/verify-links.sh:139]: actually, the indents on the stuff in the "if" below are off too.

Comment (chupman) [reply bin/verify-links.sh:139]: Sorry about that. I was testing removing a block of code and failed to notice that wild tabs appeared when I put it back.

**Pull request merged**